recipes/bob.extension/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON -B setup.py install --single-version-externally-managed --record record.txt
```

Comment:
You can remove the `build.sh` file and add `script: python -B setup.py install --single-version-externally-managed --record record.txt` field in the `build` field.